sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -2755,91 +2713,1315 @@
+            "refreshOnChange": false,
+            "customClass": "category-header-large primary-color",
+            "hidden": false,
+            "modalEdit": false,
+            "key": "reportAChangeInStudentCircumstance",
+            "tags": [],
+            "properties": {},
+            "conditional": {
+                "show": null,
+                "when": null,
+                "eq": "",
+                "json": ""
+            },
+            "customConditional": "",
+            "logic": [],
+            "attributes": {},
+            "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+            },
+            "type": "htmlelement",
+            "input": false,
+            "tableView": false,
+            "placeholder": "",
+            "prefix": "",
+            "suffix": "",
+            "multiple": false,
+            "defaultValue": null,
+            "protected": false,
+            "unique": false,
+            "persistent": false,
+            "clearOnHide": true,
+            "refreshOn": "",
+            "redrawOn": "",
+            "dataGridLabel": false,
+            "labelPosition": "top",
+            "description": "",
+            "errorLabel": "",
+            "tooltip": "",
+            "hideLabel": false,
+            "tabindex": "",
+            "disabled": false,
+            "autofocus": false,
+            "dbIndex": false,
+            "customDefaultValue": "",
+            "calculateValue": "",
+            "calculateServer": false,
+            "widget": null,
+            "validateOn": "change",
+            "validate": {
+                "required": false,
+                "custom": "",
+                "customPrivate": false,
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false
+            },
+            "allowCalculateOverride": false,
+            "encrypted": false,
+            "showCharCount": false,
+            "showWordCount": false,
+            "allowMultipleMasks": false,
+            "addons": [],
+            "id": "e15sv1b"
+        },
+        {
+            "label": "<strong>Select a change in circumstance</strong>",
+            "labelPosition": "top",
+            "labelWidth": "",
+            "labelMargin": "",
+            "optionsLabelPosition": "right",
+            "description": "",
+            "tooltip": "",
+            "customClass": "",
+            "tabindex": "",
+            "inline": false,
+            "hidden": false,
+            "hideLabel": false,
+            "autofocus": false,
+            "disabled": false,
+            "tableView": false,
+            "modalEdit": false,
+            "values": [
+                {
+                    "label": "Student went from full-time to part-time studies (vice-versa) or started a different program",
```

Comment:
Please ensure that the options have the below set:
![image](https://user-images.githubusercontent.com/61259237/163862478-e363e255-7cb8-4bcf-89e4-49981adaaa20.png)